lib/helpers/getNParents.js

```diff
@@ -7,9 +7,13 @@ export function getNParents(element, nParents = 5, parents = []) {
     return parents;
   }
 
+  remainingParents = nParents - 1
+  if (element.tagName === 'button' || element.tagName === 'a') {
+    remainingParents = 5
```

Comment:
When there is a shift error (meaning someone clicked on an `img` or `span` in front of a `button`), we end up with `button` as one of the parents instead of the base element. This produces different normalized events depending on where in the button you click, which causes problems downstream. We want to fix this in normalization by realigning the element so `button` is the base element, no matter where you click. But the `span` at the bottom took up one of the 5 parent slots, so the top level parent info is missing. This change collects the additional parent(s) so they aren't missing when we realign.

Should be safe, worst case there is a button inside of a button or something (which I don't think is even possible) and it collects too much, but eventually it will run out of parents (lines 6-8) and stop.